.github/dependabot.yml

```diff
@@ -3,7 +3,7 @@ updates:
   - package-ecosystem: github-actions
     directory: "/"
     schedule:
-      interval: daily
+      interval: weekly
```

Comment:
Do we want to group minor and patch updates also?

Reply:
oh yes, good idea
![image](https://github.com/github/github-ospo/assets/6935431/b9ae88aa-30bb-4181-95cb-8ecfc933c89c)